src/csv/index.js

```diff
@@ -200,21 +201,38 @@ const adapterFactories = {
   }),
 };
 
-function classifyFile(firstRow) {
-  if (arrayEqual(firstRow, boaCreditHeader)) return 'boaCredit';
-  if (arrayEqual(firstRow, boaBankHeader)) return 'boaBank';
-  if (arrayEqual(firstRow, venmoHeader)) return 'venmo';
-  if (arrayEqual(firstRow, cashAppHeader)) return 'cashApp';
-  if (arrayEqual(firstRow, capitalOneHeader)) return 'capitalOne';
-  if (firstRow[0] && firstRow[0].startsWith('Account Name : ')) return 'delta';
+function checkHeaderPatterns(possibleHeaderRow) {
```

Comment:
renaming to make it clear that this is just for matching header rows - everything up to and including the matched row will be _skipped_ no matter what the skipRows value